src/status_im2/contexts/wallet/account/tabs/view.cljs

```diff
@@ -1,13 +1,57 @@
 (ns status-im2.contexts.wallet.account.tabs.view
   (:require
     [quo2.core :as quo]
+    [quo2.foundations.colors :as colors]
     [react-native.core :as rn]
     [status-im2.contexts.wallet.account.style :as style]
     [status-im2.contexts.wallet.common.temp :as temp]
-    [utils.i18n :as i18n]))
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn description
+  []
+  [quo/text {:size :paragraph-2}
+   (map (fn [network]
+          ^{:key (str network)}
+          [quo/text
+           {:size   :paragraph-2
+            :weight :medium
+            :style  {:color (get colors/networks network)}}
+           (str (subs (name network) 0 3) (when (= network :arbitrum) "1") ":")])
+        temp/network-names)
+   [quo/text
+    {:size   :paragraph-2
+     :weight :monospace}
+    temp/address]])
+
+(defn about-options
```

Comment:
btw, do you know if these options are always the same or in some contexts they will be different? 🤔 

great work here btw, this opens up a lot more potential development to work on! :D

Reply:
@J-Son89 the designs does not seem to have different variations for these options

Reply:
great! 👌